core/chains/solana/chain.go

```diff
@@ -55,7 +55,7 @@ type cachedClient struct {
 
 // NewChain returns a new chain backed by node.
 func NewChain(db *sqlx.DB, ks keystore.Solana, logCfg pg.LogConfig, eb pg.EventBroadcaster, dbchain DBChain, orm ORM, lggr logger.Logger) (*chain, error) {
-	cfg := config.NewConfig(dbchain.Cfg, lggr)
+	cfg := config.NewConfig(*dbchain.Cfg, lggr)
```

Comment:
this is safe just because all the chain cfg columns have a NOT NULL?

Reply:
Yes but I don't like depending on that. Could easily check and return error or just swap in zero value in that case... :thinking: